tests/cpp17/generated_cpp17/monster_test_generated.h

```diff
@@ -821,6 +839,7 @@ struct Stat FLATBUFFERS_FINAL_CLASS : private flatbuffers::Table {
   StatT *UnPack(const flatbuffers::resolver_function_t *_resolver = nullptr) const;
   void UnPackTo(StatT *_o, const flatbuffers::resolver_function_t *_resolver = nullptr) const;
   static flatbuffers::Offset<Stat> Pack(flatbuffers::FlatBufferBuilder &_fbb, const StatT* _o, const flatbuffers::rehasher_function_t *_rehasher = nullptr);
+  struct type_traits;
```

Comment:
It would be better to move this up where the NativeTableType and Builder are.
To keep the style, it needs to rename the structure to Traits.

Reply:
Done, thanks